addons/project_issue/views/project_project_views.xml

```diff
@@ -24,7 +24,7 @@
         <field name="arch" type="xml">
             <field name="use_tasks" position="after">
                 <field name="use_issues"/>
-                <field name="issue_ids" invisible="1"/>
```

Comment:
big time. this is the difference between OCA's project kanban view loading in 5 minutes or instantly (check the issue in our internal issuetracker)